tests/unit/events/event_aoi_mapping_test.py

```diff
@@ -226,7 +226,8 @@
 
 @pytest.fixture(name='dataset')
 def dataset_fixture():
-    dataset = pm.Dataset('ToyDataset', 'toy_dataset')
+    dataset = pm.DatasetLibrary.get('ToyDataset')
```

Comment:
is this change necessary? it's a bit confusing now, because the get method actually returns a definition